manifest.json

```diff
@@ -19,7 +19,15 @@
         "run_at": "document_start"
     }],
     "options_ui": {
-        "page": "options.html"
+        "page": "options.html",
+        "browser_style": true
```

Comment:
issue: according to https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/manifest.json/options_ui#syntax it should not be set